index.php

```diff
@@ -65,7 +65,7 @@
         </div>
         <div class="box-body">
           <div class="chart">
-            <canvas id="queryOverTimeChart" style="height: 247px; width: 466px;" width="932" height="494"></canvas>
```

Comment:
This seems to me unnecessarily complicated. I substituted it with a much simpler expression.